editor/src/utils/react-performance.ts

```diff
@@ -3,213 +3,6 @@ import * as fastDeepEqual from 'fast-deep-equal'
 import { PRODUCTION_ENV } from '../common/env-vars'
 import { colorTheme } from 'uuiui'
 
-export function useHookUpdateAnalysisStrictEquals<P>(name: string, newValue: P) {
```

Comment:
these are functions to help debugging. they are not used in production code but they shouldn't be deleted!!!

Reply:
eek! fixed.